From a623b026d9c7b29471df3cae32aa1e47c575f269 Mon Sep 17 00:00:00 2001
From: Robert Joseph Evans <bobby@apache.org>
Date: Thu, 28 Jun 2012 14:23:56 +0000
Subject: [PATCH 0181/1357] HADOOP-8535. Cut hadoop build times in half (upgrade maven-compiler-plugin to 2.5.1).

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-2@1355020 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 0ac90bf2fc0ffdb160af7311c2ac610b48ebaa1c)
---
 hadoop-common-project/hadoop-common/CHANGES.txt |    2 ++
 hadoop-project/pom.xml                          |    2 +-
 2 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/CHANGES.txt b/hadoop-common-project/hadoop-common/CHANGES.txt
index 1ccc679..9f942c5 100644
--- a/hadoop-common-project/hadoop-common/CHANGES.txt
+++ b/hadoop-common-project/hadoop-common/CHANGES.txt
@@ -380,6 +380,8 @@ Release 0.23.3 - UNRELEASED
     HADOOP-8288. Remove references of mapred.child.ulimit etc. since they are
     not being used any more (Ravi Prakash via bobby)
 
+    HADOOP-8535. Cut hadoop build times in half (Job Eagles via bobby)
+
   OPTIMIZATIONS
 
   BUG FIXES
diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index e7d849b..abb9dfa 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -658,7 +658,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>2.3.2</version>
+          <version>2.5.1</version>
           <configuration>
             <source>1.6</source>
             <target>1.6</target>
-- 
1.7.0.4

